### PR TITLE
Add claim tx test

### PR DESCRIPTION
**Pull request by shanev**

### CODE REVIEW

**Comment by humanalgorithm** [comment]
Matches discussion 👍

**Review by JakeHartnell — APPROVED** [review]
utACK

File: x/claim/types/codec.go [hunk 1]
```diff
@@ -25,5 +25,14 @@ func RegisterInterfaces(registry cdctypes.InterfaceRegistry) {
 }
 
 var (
-	ModuleCdc = codec.NewProtoCodec(cdctypes.NewInterfaceRegistry())
+	amino = codec.NewLegacyAmino()
+
+	// ModuleCdc = codec.NewProtoCodec(cdctypes.NewInterfaceRegistry())
+	ModuleCdc = codec.NewAminoCodec(amino)
```
Comment (webmaster128) [inline x/claim/types/codec.go:31]: LGTM.

Could you check `x/alloc/types/codec.go` for consistency? We probably want to see the same code in both, right?

File: x/claim/module.go [hunk 2]
```diff
@@ -47,11 +47,11 @@ func (AppModuleBasic) Name() string {
 }
 
 func (AppModuleBasic) RegisterCodec(cdc *codec.LegacyAmino) {
-	types.RegisterCodec(cdc)
+	types.RegisterLegacyAminoCodec(cdc)
```
Comment (jhernandezb) [inline x/claim/module.go:50]: should this be just types.RegisterCode?

Comment (jhernandezb) [reply x/claim/module.go:50]: nvm I just saw the method was renamed

**Review by JakeHartnell — APPROVED** [review]

**Review by jhernandezb — APPROVED** [review]

**Pull request merged**